scrapper.py

```diff
@@ -44,18 +77,63 @@ def get_search_urls(self):
         pass
 
 
+class ArticleParser:
+    def __init__(self, full_url, i):
+        self.article_url = full_url
+        self.article_id = i
+        self.article = article.Article
+
+    def _fill_article_with_text(self, article_bs):
+        pass
+
+    """def parse(self):
+        web_page = requests.get(self.article_url, headers=)"""
+
+
+class HTMLParser:
+    pass
+
+
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    try:
+        os.rmdir(base_path)
+    except FileNotFoundError:
+        os.mkdir(base_path)
 
 
 def validate_config(crawler_path):
     """
     Validates given config
     """
-    pass
+    with open(crawler_path, 'r', encoding='utf-8') as config:
+        data = json.load(config)
+
+    seed_urls = data["seed_urls"]
+    max_articles = data["total_articles_to_find_and_parse"]
+
+    if not isinstance(seed_urls, list):
+        raise IncorrectURLError
+    if not seed_urls:
+        raise IncorrectURLError
+    for url in seed_urls:
+        url_validation = re.match(r'https?://', url)
+        if not url_validation:
+            raise IncorrectURLError
+
+    if not isinstance(max_articles, int) or max_articles <= 0:
```

Comment:
this is fragile
imagine max_articles is in fact a string
then you program will crash as you are going to try to compare it to integer
it is better to separate these checks

Reply:
Done